src/libaktualizr/uptane/directorrepository.cc

```diff
@@ -96,6 +96,10 @@ bool DirectorRepository::updateMeta(INvStorage& storage, Fetcher& fetcher) {
 
   // Update Director Root Metadata
   {
+    // This procedure doesn't exactly follow the spec:
+    // https://uptane.github.io/uptane-standard/uptane-standard.html#check_root At first, it should try to download N+1
+    // version of the Root metadata file instead of downloading the "latest" one how do we know which version is the
+    // latest and why just root.json is downloaded ?
```

Comment:
`root.json` without a number is guaranteed to be the latest in our scheme. The design is to fetch the latest, and if it matches what we already have stored, we're good. If not, then we have to go fetch the missing ones by name/number until we catch up. Might be worth a renewed discussion on what the best method is; I don't see the harm in our method.

Reply:
Thanks for explanation.